hipchat.go

```diff
@@ -201,9 +204,14 @@ func (c *Client) listen() {
 				continue
 			}
 
+			// parse time
+			time_pieces := strings.Split(attr["ts"], ".")
+			sec, _ := strconv.ParseInt(time_pieces[0], 10, 64)
```

Comment:
please check the error being returned instead of throwing it away. and what do you think about converting the `ts[1]` from milliseconds to nanoseconds, eg:

```go
nsec, err := strconvParseInt(ts[1], 10, 64)
if err != nil {
  // ...
}
nsec *= int64(1e6)
```